dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/imports/preheat/supplier/strategy/TrackerEntityStrategyTest.java

```diff
@@ -1,73 +0,0 @@
-/*
- * Copyright (c) 2004-2022, University of Oslo
```

Comment:
Is the strategy tested somewhere?

Reply:
Not a unit test but it is covered by several integration tests.
A unit test now would be weird as we should assert on the SQL to verify that the strategy is working.